app/.nvmrc

```diff
@@ -1 +1 @@
-18.17.0
\ No newline at end of file
+20.10.0
```

Comment:
Had to bump this as well. While bumping the Forge runtime version fro 14 > 18 solved the M1 issue I was facing, it surfaced a bug with Node 18 and DNS resolution where it ipv6 instead of ipv4 and it resloves localhost to ::1 instead of 127.0.0.1.